roles/container_runtime/tasks/package_crio.yml

```diff
@@ -57,6 +57,12 @@
     src: crio.conf.j2
     backup: yes
 
+- name: Update crictl.yaml runtime-endpoint
+  yedit:
+    src: /etc/crictl.yaml
+    key: runtime-endpoint
+    value: "unix://{{ openshift_crio_var_sock }}"
```

Comment:
```suggestion
    value: "unix://{{ openshift_crio_var_sock }}"
  when: not openshift_crio_var_sock.startswith('unix://')
```

Reply:
@vrutkovs could we do something like this instead?

```yaml
set_fact:
  crio_var_sock: "{% if not openshift_crio_var_sock.startswith('unix://') %}unix://{{ openshift_crio_var_sock}}{% else %}{{ openshift_crio_var_sock}}{% endif %}"
```
That way if `openshift_crio_var_sock` was set and started with `unix://` the task wouldn't be skipped.

Reply:
That looks a bit too complex, but might work, yes